frame/lending/src/repay_borrow.rs

```diff
@@ -9,6 +9,7 @@ use crate::Config;
 /// Release given `amount` of `debt_token` from `beneficiary_account`, transfer `amount` from
```

Comment:
the contents of this file should be moved to under the helpers folder

Reply:
sure